enterprise/backend/src/metabase_enterprise/semantic_search/index_metadata.clj

```diff
@@ -322,10 +348,11 @@
                                              :table_name        index-table-name
                                              :index_version     index-version
                                              :index_created_at  [:now]}])
-                       (sql.helpers/returning :id)
+                       (sql.helpers/returning :id :index_created_at)
                        (sql/format :quoted true))
-        {:keys [id]} (jdbc/execute-one! pgvector insert-sql {:builder-fn jdbc.rs/as-unqualified-lower-maps})]
-    id))
+        {:keys [id index_created_at]}
+        (jdbc/execute-one! pgvector insert-sql {:builder-fn jdbc.rs/as-unqualified-lower-maps})]
+    (assoc index :id id :index-created-at index_created_at)))
```

Comment:
My gut is saying: 
- Keep essential index parameters seperate from states/modes/instantiation in a particular metadata/schema/db/conn/tx whatever

Another daemon is saying:
Who cares its just a map return man, call index index-spec (good one!), index can be relaxed and we all get the data we want and need without extra stuff or ideas

Would you be opposed to a basic multiple return {:id, :index-created-at} from here and not opening the door on adding metadata stuff to what we call 'index'? Not 100% on this - might be easier on brain to not open that door rather than bike shed / think too hard. Selfishly thinking about having to self review DLQ again with this in change in mind.

Reply:
IDK man, I'm probably bike shedding, this might be better anyway - it probably does not matter, I'll leave it up to you (conflicted!)

Reply:
I think eventually a split between index-spec/index-instance would make sense, and :id/:index-created-at or whatever else from metadata-row would end up in index-instance 

In practice, the need for carrying this info was to prepare `init!` for the eventual rework of `reindex!`, but given the potential change in semantics of reindex! (+ our switch of teams), I'd rather back this out instead of leaving breadcrumbs that may not be necessary to whoever ends up picking this up next.